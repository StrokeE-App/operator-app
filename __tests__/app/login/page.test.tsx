import {render} from '@testing-library/react';
import LoginPage from '@/app/login/page';

describe('LoginPage', () => {
	it('matches snapshot', () => {
		const {container} = render(<LoginPage />);
		expect(container).toMatchSnapshot();
	});
});
