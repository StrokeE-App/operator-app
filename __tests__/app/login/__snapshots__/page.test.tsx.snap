// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage matches snapshot 1`] = `
<div>
  <main
    class="flex min-h-screen flex-col items-center justify-center p-8 gap-8"
  >
    <div
      class="flex flex-col items-center gap-2"
    >
      <svg
        class="lucide lucide-siren"
        fill="none"
        height="64"
        stroke="#d63a3a"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="2"
        viewBox="0 0 24 24"
        width="64"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M7 18v-6a5 5 0 1 1 10 0v6"
        />
        <path
          d="M5 21a1 1 0 0 0 1 1h12a1 1 0 0 0 1-1v-1a2 2 0 0 0-2-2H7a2 2 0 0 0-2 2z"
        />
        <path
          d="M21 12h1"
        />
        <path
          d="M18.5 4.5 18 5"
        />
        <path
          d="M2 12h1"
        />
        <path
          d="M12 2v1"
        />
        <path
          d="m4.929 4.929.707.707"
        />
        <path
          d="M12 12v6"
        />
      </svg>
      <h1
        class="text-red-600 text-2xl font-bold tracking-wider"
      >
        STROKEE
      </h1>
    </div>
    <div
      class="w-6 h-6 border-2 border-customRed border-t-transparent rounded-full animate-spin"
    />
  </main>
</div>
`;
